README.md

```diff
@@ -73,6 +73,19 @@ desirable but not yet implemented.
 Execute `cargo test` to run all tests, or check out the latest
 [CI run](https://cloud.drone.io/erikgrinaker/toydb).
 
+## Debugging
+
+VS Code provides a very intuitive environment for debugging toyDB. Follow these steps to set it up:
+1. Install [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) extension
+1. Go to "Run and Debug" tab and initialize launch.json, here is my autogenerated [`launch.json`](https://gist.github.com/leotoff/78e20cb1948e8bb20631271576730865) for reference
+
+The above lets debug `toysql` (from "Run and Debug" tab) and unit tests (using a popup in test files). Debugging the engine is a bit more involved:
+1. Add config arg `"-c=${workspaceFolder}/config/toydb.yaml"` to "Debug executable 'toydb'" configuration in `launch.json`, see [my configuration](https://gist.github.com/leotoff/b940df1b947b7d4e6fd42bd057943fe3) for reference
+1. Create a data directory `sudo mkdir /var/lib/toydb && sudo chown $(whoami) /var/lib/toydb` (the path is specified in `config/toydb.yaml`)
```

Comment:
Maybe we should just change the default to `./data`. `/var/lib/toydb` is rarely useful.

Reply:
sounds good. do you want me to modify `config/toydb.yaml` in this or a separate PR? if in a separate PR, I'll modify README.md there as well (`/var/lib/toydb` -> `./data`). Let me know how to approach this.

Reply:
Separate PR would be great, thanks!